networking/metallb/metallb-upgrading-operator.adoc

```diff
@@ -6,11 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The automatic upgrade procedure does not work as expected from {product-title} 4.10 and earlier. A summary of the upgrade procedure is as follows:
+The automatic upgrade procedure does not work as expected from {product-title} 4.11 and earlier. A summary of the upgrade procedure is as follows:
```

Comment:
@xenolinux Upgrade from 4.10 -> 4.11 or 4.10 -> 4.12 needs 4.10 to be removed and later version be installed. 4.11->4.12 should be automatically upgraded.

Reply:
So that means we have to revert the changes?